code/game/machinery/_machinery.dm

```diff
@@ -189,7 +190,7 @@ Class Procs:
 	density = TRUE
 	if(!target)
 		for(var/am in loc)
-			if(!is_type_in_typecache(am, occupant_typecache))
+			if(!is_type_in_typecache(am, (occupant_typecache || GLOB.typecache_living)))
```

Comment:
wouldn't this allow all living things in thigns that aren't meant to have all living things in it?

Reply:
No because || operator is short circuited